vendor/vtd-api/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class VtdApiConan(ConanFile):
+    name = "vtd-api"
+    version = "2.2.0"
+    license = "Proprietary"
+    url = "https://vires.mscsoftware.com"
+    no_copy_source = True
+    description = "Include binary files in C/C++"
+    topics = ("simulator", "vires")
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    generators = "cmake"
+
+    _archive_base = "../vtd/vtd.2.2.0.Base.20181231.tgz"
+    _root_dir = "VTD.2.2"
+
+    def export_sources(self):
+        tools.untargz(
+            self._archive_base,
+            self.export_sources_folder,
+            pattern=Path(self._root_dir) / "Develop" / "*",
```

Comment:
Similar error as in the `vtd/conanfile.py`. This works for me:
`pattern="{}".format(Path(self._root_dir) / "Develop" / "*"),`